test/src/e2e_vm_tests/test_programs/should_pass/stdlib/exponentiation_test/src/main.sw

```diff
@@ -0,0 +1,59 @@
+script;
+
+use std::{ assert::assert, math::* };
```

Comment:
This file just needs formatting. Looks good otherwise!

Reply:
Fixed in [e1f8702](https://github.com/FuelLabs/sway/pull/1744/commits/e1f8702185aeedf3300f4b3cb107eb20acc6cedc)